build/rule-generator/get-files-metadata.js

```diff
@@ -127,7 +132,10 @@ const getCheckJsFileMeta = (name, dir) => {
 		name: `${name}.js`,
 		content: `
 		// TODO: Logic for check
-		return true;
+		function evaluate(node) {
```

Comment:
This should also be named based on the name of the file so it's not just `evaluate` for every new rule.